library/pod-security-policy/users/template.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: templates.gatekeeper.sh/v1beta1
+kind: ConstraintTemplate
+metadata:
+  name: k8spspallowedusers
+spec:
+  crd:
+    spec:
+      names:
+        kind: K8sPSPAllowedUsers
+        listKind: K8sPSPAllowedUsersList
+        plural: k8spspallowedusers
+        singular: k8spspallowedusers
+      validation:
+        openAPIV3Schema:
+          properties:
+            runAsUser:
+              type: integer
```

Comment:
`runAsUser` parameter should contain a string type `rule` and an array type`ranges` with type integer `min` and `max`

Reply:
Thanks for catching this Rita. I copied the structure of fsgroup which is identical.